cypress/integration/admin.spec.js

```diff
@@ -0,0 +1,44 @@
+const { join } = require('path');
```

Comment:
"join" loses its context when it's destructured from `path`—it's not clear what's being joined.

```suggestion
const path = require('path'); // and then just use path.join() elsewhere
```

```suggestion
const { join: joinPathSegments } = require('path');
```